vpp/vaapipostprocess_scaler.cpp

```diff
@@ -59,9 +62,29 @@ VAProcColorStandardType fourccToColorStandard(uint32_t fourcc)
     return VAProcColorStandardBT601;
 }
 
+VaapiPostProcessScaler::VaapiPostProcessScaler()
```

Comment:
It's same process. It share most of code and our driver can do deinterlace and scale in same time.